dot/import_test.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package dot
+
+import (
+	"encoding/json"
+	"errors"
+	"math/big"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupStateFile(t *testing.T) string {
+	t.Helper()
+
+	const filename = "../lib/runtime/test_data/kusama/block1482002.out"
```

Comment:
```suggestion
	rootPath, err := GetProjectRootPath()
	require.NoError(t, err)
	filename := filepath.Join(rootPath, "lib/runtime/test_data/kusama/block1482002.out")
```

so it works everywhere

Reply:
Updated.